worldtimebuddy/cli.py

```diff
@@ -21,9 +37,27 @@ def cli(format, major, timezone, list_timezones):
     now = datetime.datetime.now(pytz.utc)
     
     if timezone:
+        # Map short timezone code to full name if applicable
+        timezone = timezone_codes.get(timezone, timezone)
+        
         try:
             tz = pytz.timezone(timezone)
             time = now.astimezone(tz)
+            
+            if delta:
+                if 'hr' in delta:
+                    hours = int(delta.replace('hr', ''))
+                    time += datetime.timedelta(hours=hours)
+                elif 'min' in delta:
+                    minutes = int(delta.replace('min', ''))
+                    time += datetime.timedelta(minutes=minutes)
+                elif 'day' in delta:
+                    days = int(delta.replace('day', ''))
+                    time += datetime.timedelta(days=days)
+                else:
+                    click.echo("Error: Invalid time format. Use +2hr, -30min, -1day, etc.")
+                    return
+
```

Comment:
Consider this to add in utils

Reply:
can you kindly elaboraite on this? thank you.

Reply:
Consider moving the block 

```
if 'hr' in delta:
    hours = int(delta.replace('hr', ''))
    time += datetime.timedelta(hours=hours)
elif 'min' in delta:
    minutes = int(delta.replace('min', ''))
    time += datetime.timedelta(minutes=minutes)
elif 'day' in delta:
    days = int(delta.replace('day', ''))
    time += datetime.timedelta(days=days)
```
into some util function in utils.py file.